examples/ffcv_enable.py

```diff
@@ -0,0 +1,147 @@
+"""
+This example shows how to use FFCV data loading system in Avalanche.
+"""
```

Comment:
suggestion: make a ffcv folder with a README.md explaining the purpose of each example. Then we can just refer users to that single file.